packages/algorithm/lib/variants/bgoncharuck/methods_userpollcalc.dart

```diff
@@ -0,0 +1,125 @@
+import 'package:meta/meta.dart';
+import 'package:prophecies/prophecies.dart';
+import 'package:prophecy_model/prophecy_model.dart';
+import 'package:poll_model/poll_model.dart';
+
+ProphecyEntity changePartOfBase(
```

Comment:
Naming makes it harder to understand. 
Methods are not self-explanatory, as well as some of the params.

For ex. `bySign` -- in order to understand what it is, I have to go back in the place where `changePartOfBase` is called
```
  /// here we choose a position by the index
  /// of given poll value - 1
  final position = bySign[(pollValue >= 3.0)
```
This one should have helped, but it is still not clear what `bySign` is

Reply:
I dont know how to name it.
it is value for astrologic sign, for a specific poll and its voted value.